src/components/ProjectsPage.js

```diff
@@ -1,83 +1,117 @@
-                 {id: 4, title: "Project Title 5", name: "Organization Name 5"}]
 
 class ProjectsPage extends React.Component {
+  state = {
+    selected: -1
+  }
+
+  render() {
+    return (
+      <Fragment>
+        { this.state.selected === -1 &&
+        <Container className="pg-container">
+          <Row className="d-flex justify-content-center">
+            <Row className="d-flex justify-content-left">
+              {projects.map(project =>
+                <Col lg={4}>
+                  <Card className="broad card-btn" onClick={() => this.setState({selected: project.id})}>
+                    <Body>
+                      <Title> {project.title} </Title>
+                      <Subtitle className="mb-2 text-muted"> {project.company} </Subtitle>
+                      <Button className="urgent-tag"> Urgent </Button>
+                      <Text class="card-text">Project description. We are looking for a group of
+                                                   people to help us put together a simple website for
+                                                   people to email their state reps.
+                      </Text>
+                      <Link href="#" className="float-right" onClick={() => this.setState({selected: project.id})}>More details...</Link>
+                    </Body>
+                  </Card>
+                </Col>
+              )}
+            </Row>
+          </Row>
+        </Container>
+        }
+
+
+
+
+
+        { this.state.selected !== -1 &&
+        <Container className="pg-container center-div">
+          <Row>
+            <Col lg={4} className="overflow-auto scrolling project-col d-none d-lg-block">
+              {projects.map(project =>
+                <Card className = {`preview-card ${(this.state.selected !== project.id) ? "card-btn" : "selected-btn"}`}
+                      onClick={() => this.setState({selected: project.id})}>
+                  <Body>
+                    <Title> {project.title} </Title>
+                    <Subtitle className="mb-2 text-muted"> {project.company} </Subtitle>
+                    <Button className="urgent-tag"> Urgent </Button>
+                    <Text>Project description. We are looking for a group of
+                               people to help us put together a simple website for
+                               people to email their state reps.
+                    </Text>
+                    <Link href="#" className="float-right" onClick={() => this.setState({selected: project.id})}>More details...</Link>
+                    </Body>
+                  </Card>
+              )}
+            </Col>
+
+            <Col lg={7} md={12} className="">
+              <Card className="description-card">
+                <Body>
+                  <div class="d-flex justify-content-lg-end justify-content-md-start">
+                    <Button className="close-btn mb-4 mb-lg-0" onClick={() => this.setState({selected: -1})}> Close </Button>
```

Comment:
Could we change this to just an X instead of a button that says close? Less to read

Reply:
👍